src/checkUrl.js

```diff
@@ -0,0 +1,36 @@
+const cases = [
+  'SNAKE',
+  'KEBAB',
+  'CAMEL',
+  'PASCAL',
+  'UPPER',
+];
+
+const checkUrl = (text, toCase) => {
+  const errors = [];
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. '
+      + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: '"toCase" query param is required. '
+      + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!cases.includes(toCase) && toCase) {
+    errors.push({
+      message: 'This case is not supported. '
+      + 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    });
```

Comment:
You have the variable `cases`, use values from it to have always up to date errors
```suggestion
    errors.push({
      message: `This case is not supported. Available cases: ${cases.join(', ')}.`,
    });
```